assets/aml-benchmark/dev_conda_env.yaml

```diff
@@ -0,0 +1,19 @@
+channels:
```

Comment:
nit: add under tests?

Reply:
will let it be as it contains both src and test requirements. It is not only helpful for running tests, but also for running the scripts locally.